.github/workflows/post-release.yml

```diff
@@ -36,6 +36,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Wait for Zenodo webhook
+        run: sleep 120
```

Comment:
Hardcoded sleep seems like an issue- is there a way to link the action as a wait instead?

Reply:
Nope. That's out of our control, depends on how much time requires Zenodo to make a new release on their server. 2 mins seems enough.

Reply:
What happens if this wait time is exceeded?

Reply:
A pull request is opened just with changes in CHANGELOG.rst, CITATION.cff will not updated because the release was the same at that moment the job started.

Reply:
This is done, the warning looks like this: https://github.com/tardis-sn/tardis/pull/2018